src/StatsigClient.ts

```diff
@@ -154,20 +150,27 @@ export default class StatsigClient {
    * @returns {boolean} - value of a gate for the user. Gates are "off" (return false) by default
    * @throws Error if initialize() is not called first, or gateName is not a string
    */
-  public checkGate(gateName: string): boolean {
-    return this._checkGateImpl(gateName, 'checkGate');
+  public checkGate(user: StatsigUser | null, gateName: string): boolean {
```

Comment:
should we be allowing null user?

Reply:
haha was wondering if youd noticed. Still tbd on how we handle. Likely not